ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -1,6 +1,23 @@
 <div class="container-fluid wu-frame">
     <form #projectForm="ngForm" (ngSubmit)="save()" class="form-horizontal wizard-form">
 
+        <style>
```

Comment:
I was also trying the Angular validators but this was easier to code. Just CSS and HTML, no calls, no unnecessary model properties.